go.mod

```diff
@@ -95,4 +95,4 @@ replace (
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.17.1
 	vbom.ml/util => github.com/fvbommel/util v0.0.0-20180919145318-efcd4e0f9787
 
-)
\ No newline at end of file
+)
```

Comment:
Thank you! I thought I was the only one getting this annoying change :roll_eyes: